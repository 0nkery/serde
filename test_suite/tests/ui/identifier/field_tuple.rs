#[macro_use]
extern crate serde_derive;

#[derive(Deserialize)]
#[serde(field_identifier)]
enum F {
    A,
    B(u8, u8),
}

fn main() {}
