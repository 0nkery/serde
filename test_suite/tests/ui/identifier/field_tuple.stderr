error: field_identifier may only contain unit variants
 --> $DIR/field_tuple.rs:4:10
  |
4 | #[derive(Deserialize)]
  |          ^^^^^^^^^^^

error: aborting due to previous error

